packages/salesforcedx-vscode-core/src/orgBrowser/nodeTypes.ts

```diff
@@ -134,6 +134,7 @@ export class BrowserNode extends vscode.TreeItem
     switch (this.type) {
       case NodeType.MetadataType:
         return RetrieveDescriberFactory.createTypeNodeDescriber(this);
+      case NodeType.Folder:
```

Comment:
What happens when we do a retrieve on a folder that is not sobject?

Reply:
It runs the command successfully.
<img width="519" alt="Screen Shot 2021-12-15 at 10 08 21 AM" src="https://user-images.githubusercontent.com/72183558/146241672-90020700-be50-4708-b503-4415d22ade1f.png">
.